deno-app/main.ts

```diff
@@ -0,0 +1,52 @@
+import { serve } from "https://deno.land/std@0.173.0/http/server.ts";
+import { Deta } from "https://esm.sh/deta@1.1.0";
```

Comment:
why not?
```suggestion
import Deta from "npm:deta";
```
ref: https://github.com/mikBighne98/deno-on-deta-space/blob/8b36bfd88051551984dc7e11d053404914fad493/src/deno-crud/routes/base.ts

Reply:
why not even directly import the service classes.
```suggestion
import Base from "npm:deta";
```

Reply:
Haven't tested with the npm version yet, will try now. Reason I import `Deta` is to pass it the value of the env variable, since for some reason it doesn't get it itself. Raises "no project key defined" when deployed on Space.

Reply:
npm specifiers are not supported by the `compile` subcommand yet.